api/config.py

```diff
@@ -323,3 +326,11 @@ def mongo_pipeline(table, pipeline):
 
 def get_auth(auth_type):
     return get_config()['auth'][auth_type]
+
+
+# Storage configuration
+fs = open_fs(__config['persistent']['fs_url'])
+signed_url_available = ['GCSFS']
```

Comment:
During the design phase we discussed that storages should be able to tell about themselves, whether they support signed URLs or not. It's not a hard requirement, especially if it's not easy to solve, but I would prefer that to having this config variable.